src/Raven.Server/Web/System/AdminMonitoringHandler.cs

```diff
@@ -198,6 +198,15 @@ private DiskMetrics GetDiskMetrics()
                     result.TotalFreeSpaceInMb = totalFreeMb;
                     result.RemainingStorageSpacePercentage = percentage;
                 }
+                var diskStatsResult = Server.DiskStatsGetter.Get(ServerStore._env.Options.DriveInfoByPath?.Value.BasePath.DriveName);
+                if (diskStatsResult != null)
+                {
+                    result.IoReadOperations = diskStatsResult.IoReadOperations;
+                    result.IoWriteOperations = diskStatsResult.IoWriteOperations;
+                    result.ReadThroughputInKb = diskStatsResult.ReadThroughput.GetValue(SizeUnit.Kilobytes);
```

Comment:
That GetValue call keeps coming back in the code every now and then. Maybe an extension method would come in handy? e.g. `diskStatsResult.GetReadThroughputInKilobytes()`

Reply:
I agree one code path for making a decision is in general better
but I fill like maybe here is too much.
Even though, I added an extension method.